8Knot/pages/affiliation/visualizations/org_core_contributors.py

```diff
@@ -48,99 +90,114 @@
-                                    ),
-                                    className="me-2",
-                                    width=2,
+                                    [
+                                        dbc.Label(
+                                            "Core Contributors Required:",
+                                            html_for=f"contributors-required-{PAGE}-{VIZ_ID}",
+                                            style={"marginBottom": "8px", "fontSize": "14px"},
+                                        ),
+                                        dbc.Input(
+                                            id=f"contributors-required-{PAGE}-{VIZ_ID}",
+                                            type="number",
+                                            min=1,
+                                            max=50,
+                                            step=1,
+                                            value=3,
+                                            size="sm",
+                                            className="dark-input",
+                                            style={"width": "80px"},
+                                        ),
+                                    ],
+                                    width="auto",
+                                    className="me-4",
                                 ),
-                            ],
-                            align="center",
-                        ),
-                        dbc.Row(
-                            [
                                 dbc.Col(
-                                    dcc.DatePickerRange(
-                                        id=f"date-picker-range-{PAGE}-{VIZ_ID}",
-                                        min_date_allowed=dt.date(2005, 1, 1),
-                                        max_date_allowed=dt.date.today(),
-                                        initial_visible_month=dt.date(dt.date.today().year, 1, 1),
-                                        clearable=True,
-                                    ),
+                                    [
+                                        dbc.Label("Email Filters:", style={"marginBottom": "8px", "fontSize": "14px"}),
+                                        dbc.Checklist(
+                                            id=f"email-filter-{PAGE}-{VIZ_ID}",
+                                            options=[
+                                                {
+                                                    "label": "Exclude Gmail",
+                                                    "value": "gmail",
+                                                },
+                                                {
+                                                    "label": "Exclude GitHub",
+                                                    "value": "github",
+                                                },
+                                            ],
```

Comment:
It seems like this email filters module is also duplicated in `org_associated_activity.py` - is there a way to factor this (seemingly commonly used UI element) out into its own file or something and include it in both places?